MaterialComponents.podspec

```diff
@@ -211,6 +211,7 @@ Pod::Spec.new do |mdc|
     component.dependency "MaterialComponents/ShadowLayer"
     component.dependency "MaterialComponents/private/Icons/ic_check_circle"
     component.dependency "MaterialComponents/private/Math"
+    component.dependency "MaterialComponents/private/Shapes"
```

Comment:
The summary doesn't have much context - why is this needed? I don't see any code changes.

Reply:
Updated the PR description.